app/models/nic/base.rb

```diff
@@ -7,13 +7,10 @@ class Base < ActiveRecord::Base
     self.table_name = 'nics'
 
     validates_lengths_from_database
-    attr_accessible :host_id, :host,
-                    :mac, :name, :type,
-                    :provider, :username, :password,
-                    :identifier, :virtual, :link, :tag, :attached_to,
-                    :managed, :bond_options, :attached_devices, :mode,
-                    :primary, :provision, :compute_attributes,
-                    :_destroy # used for nested_attributes
+    attr_accessible :attached_to, :attached_devices, :bond_options,
+      :compute_attributes, :host_id, :host, :identifier, :ip, :link, :mac,
+      :managed, :mode, :name, :password, :primary, :provider, :provision, :type,
+      :tag, :username, :virtual, :_destroy # used for nested_attributes
```

Comment:
the comment applies for `:_destroy` only which is not clear now, anyway I think you can remove it, I think everyone knows that...